src/modules/bunnymq.js

```diff
@@ -49,10 +49,14 @@ module.exports = (connection) => {
     instance.connection = connection;
   }
   return {
-    consume: instance.consume.bind(instance),
-    subscribe: instance.subscribe.bind(instance),
-    produce: instance.produce.bind(instance),
-    publish: instance.publish.bind(instance),
-    connection: instance.connection
+    connection: instance.connection,
+    consumer: {
+      consume: instance.consume.bind(instance),
+      subscribe: instance.subscribe.bind(instance)
+    },
+    producer: {
+      produce: instance.produce.bind(instance),
+      publish: instance.publish.bind(instance)
```

Comment:
I think we need to support both for backward compatibility and to satisfy the need by the issue you mentioned, @ky23 ?